<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FORM_CanBusQuery</class>
 <widget class="QWidget" name="FORM_CanBusQuery">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*background-image: url(:/pic_de260/bg_01.png);*/
QLabel{
color: rgb(15, 175, 252);
font: 17pt &quot;WenQuanYi Zen Hei&quot;;
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>50</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>TakaoPGothic</family>
     <pointsize>30</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(252, 233, 79);
font: 30pt &quot;TakaoPGothic&quot;;</string>
   </property>
   <property name="text">
    <string>总线状态</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1290</x>
     <y>50</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Zen Hei</family>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/ico_260/914.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="text1">
   <property name="geometry">
    <rect>
     <x>951</x>
     <y>156</y>
     <width>300</width>
     <height>29</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Zen Hei</family>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>主控制器CAN</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="light_1">
   <property name="geometry">
    <rect>
     <x>1257</x>
     <y>156</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/pic_de260/ico_27.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="text3">
   <property name="geometry">
    <rect>
     <x>951</x>
     <y>207</y>
     <width>300</width>
     <height>29</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Zen Hei</family>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>液压控制器CAN</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="light_3">
   <property name="geometry">
    <rect>
     <x>1257</x>
     <y>207</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/pic_de260/ico_27.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="text2">
   <property name="geometry">
    <rect>
     <x>951</x>
     <y>257</y>
     <width>300</width>
     <height>29</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Zen Hei</family>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>发动机控制器CAN</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="light_2">
   <property name="geometry">
    <rect>
     <x>1257</x>
     <y>257</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/pic_de260/ico_27.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="text4">
   <property name="geometry">
    <rect>
     <x>951</x>
     <y>308</y>
     <width>300</width>
     <height>29</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Zen Hei</family>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>10.4仪表CAN</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="light_4">
   <property name="geometry">
    <rect>
     <x>1257</x>
     <y>308</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/pic_de260/ico_27.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="text5">
   <property name="geometry">
    <rect>
     <x>951</x>
     <y>359</y>
     <width>300</width>
     <height>29</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Zen Hei</family>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>总线键盘CAN</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="light_5">
   <property name="geometry">
    <rect>
     <x>1257</x>
     <y>359</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/pic_de260/ico_27.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="text5_2">
   <property name="geometry">
    <rect>
     <x>951</x>
     <y>410</y>
     <width>300</width>
     <height>29</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Zen Hei</family>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>GE系统CAN</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="light_6">
   <property name="geometry">
    <rect>
     <x>1257</x>
     <y>410</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/pic_de260/ico_27.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="text5_3">
   <property name="geometry">
    <rect>
     <x>951</x>
     <y>460</y>
     <width>300</width>
     <height>29</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Zen Hei</family>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>称重系统CAN</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="light_7">
   <property name="geometry">
    <rect>
     <x>1257</x>
     <y>460</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/pic_de260/ico_27.png</pixmap>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
